server/views/pages/teamOverview.njk

```diff
@@ -157,19 +164,19 @@
           <div class="govuk-grid-column-one-half">
             <h4 class="govuk-heading-s">Recommended Channels</h4>
             <ul class="govuk-list govuk-list--bullet">
-              <li><strong>{{ channelRecommendations.suggestedChannels.nonProd }}</strong> - For dev and preprod alerts</li>
-              <li><strong>{{ channelRecommendations.suggestedChannels.prod }}</strong> - For production alerts</li>
-              <li class="govuk-hint govuk-body-s">Alternative: <strong>{{ channelRecommendations.suggestedChannels.prodAlt }}</strong> - Shorter prod channel name</li>
+              <li><strong>{{ team.channelRecommendations.suggestedChannels.nonProd }}</strong> - For dev and preprod alerts</li>
+              <li><strong>{{ team.channelRecommendations.suggestedChannels.prod }}</strong> - For production alerts</li>
+              <li class="govuk-hint govuk-body-s">Alternative: <strong>{{ team.channelRecommendations.suggestedChannels.prodAlt }}</strong> - Shorter prod channel name</li>
             </ul>
           </div>
           <div class="govuk-grid-column-one-half">
             <h4 class="govuk-heading-s">Channel Structure</h4>
-            <pre class="govuk-body-s" style="background-color: #f3f2f1; padding: 15px; border-radius: 4px; font-family: monospace;">{% for line in channelTree %}{{ line }}
+            <pre class="govuk-body-s" style="background-color: #f3f2f1; padding: 15px; border-radius: 4px; font-family: monospace;">{% for line in team.channelTree %}{{ line }}
```

Comment:
Don't use inline styles. This needs turning into a class and put in the css file. It's impossible to override inline styles in the css file, plus you're sending this data every time the page is sent, in the css file it's cached and only the class name is here.